nucypher/cli/options.py

```diff
@@ -190,3 +190,4 @@ def process_middleware(mock_networking):
     process_middleware,
     mock_networking=click.option('-Z', '--mock-networking', help="Use in-memory transport instead of networking", count=True),
     )
+option_signer_uri = click.option('--signer', 'signer_uri', '-S', default=None, type=str)
```

Comment:
Do we need any help text?

```suggestion
option_signer_uri = click.option('--signer', 'signer_uri', '-S', help="Signer's URI e.g. '/path/to/clef.ipc'", default=None, type=str)
```

Side note: does the signer URI allow for different URI schemes eg. `file://` or `ipc://` like we do for `--provider`? Does it matter?

Reply:
At the moment, only file-based IPC schemes are implemented.